### PR TITLE
Code review fixes

### PR DESCRIPTION
**Pull request by SnirShechter**

### CODE REVIEW

File: sdk/tests/test_xgboost.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 @pytest.fixture
 def sample_data() -> xgboost.DMatrix:
-    sample_data_path = Path(Path(__file__).parent, "data", "xgboost.txt")
+    sample_data_path = Path(__file__).parent / Path("data", "xgboost.txt")
```
Comment (yonbek) [inline sdk/tests/test_xgboost.py:16]: What about test_lightgbm?

File: sdk/src/mlnotify/logger.py [hunk 2]
```diff
@@ -2,24 +2,39 @@
...
+    )
+    is_logger_initiated = True
+    logger = logging.getLogger(LOGGER_NAME)
 
-logger.debug(f"Logger is set to level {logger.level}")
+    logger.debug(f"Logger is set to level {logger.level}")
+    return logger
+
+
+def get_logger():
```
Comment (yonbek) [inline sdk/src/mlnotify/logger.py:33]: There's no need for lazy initialization, I just wanted the logic moved to a function to make it cleaner and easier to mock in tests

**Review by yonbek — CHANGES_REQUESTED** [review]

File: sdk/tests/test_xgboost.py (hunk 1 above)
Comment (SnirShechter) [reply sdk/tests/test_xgboost.py:16]: Done.

File: sdk/src/mlnotify/logger.py (hunk 2 above)
Comment (SnirShechter) [reply sdk/src/mlnotify/logger.py:33]: Done.

**Pull request merged**